web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte

```diff
@@ -215,13 +215,7 @@
           start,
           end,
         });
-
-        // hack to prevent race condition in URL proto changes
-        setTimeout(() => {
-          metricsExplorerStore.setSelectedScrubRange(metricViewName, undefined);
-        }, 50);
-      }
-      if (
```

Comment:
Do we have to remove this? I can still see scrub being shown even after we zoom in

Reply:
Lets fix this on main and unblock you. We should not need the timeout.